hadoop-hdds/docs/content/design/s3-performance.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Persistent OM connection for S3 gateway
+summary: Use per-request authentication and persistent connections between S3g and OM
```

Comment:
Can we add the word proposal here in the title/summary?

Reply:
Sure. Added in 9c6ce9158.